### ISSUE TITLE
focus input

### ISSUE THREAD

**Issue by kalken** [marcosmoura/vue-material#594]
First off, thanks for a great library!

I'm trying to focus an input-box when a page is rendered, but even when following the instructions i get from: https://vuejs.org/v2/guide/custom-directive.html i run in to some problems.

Code is here but its a bit tricky to show the bug because of the other things that happens on this page: http://codepen.io/anon/pen/zZoNNa

if you change the console.log text and quickly put the mousepointer on the page render area you can reproduce the problem i'm having.

In short the problem is that the "focus"-function does work, but it's not the same thing as if i click on the input box. When i run focus() on the input element its still greyed (not active),  and the label does not go up over the input field. Do i need to trigger something else in the input-container to get the same behaviour as if i'm clicking in an input field or is this a bug?

**Comment by kalken** [comment]
Ok, i looked more into this and the problem seems to be that isFocused on the parent container never gets set.
```
  directives: {
    focus: {
      inserted: (el,binding,vnode) => {
        el.focus()
        vnode.child.$parent.isFocused = true
      }
    }
  },
```
that works... Still dont know if this is a bug or not

**Comment by tsunamilx** [comment]
Didn't work for me.
I have to set timeout in order to make it work:
```javascript
setTimeout(() => {this.$refs.focusable.$el.focus();}, 500);
```
```html
<md-input ref="focusable"></md-input>
```

**Comment by marcosmoura** [comment]
Just use this.$nextTick to achieve this.
